sqrl-execute/sqrl-execute-flink/sqrl-execute-flink-function/src/main/java/com/datasqrl/TimeFunctions.java

```diff
@@ -372,15 +372,13 @@ public Instant eval(String s) {
     }
 
     public Instant eval(String s, String format) {
-      DateTimeFormatter formatter = DateTimeFormatter.ofPattern(format, Locale.US);
-      try {
-        return LocalDateTime.parse(s, formatter)
-            .atZone(ZoneId.systemDefault())
-            .toInstant();
-      } catch (Exception e) {
-        log.warn(e.getMessage());
-        return null;
-      }
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(format, Locale.US);
+        try {
+            return ZonedDateTime.parse(s, formatter).toInstant();
```

Comment:
Thanks for the contribution! We're all PST guys so this feedback has been very important to us.

Earlier we made the decision to normalize our dates for stream processing engines. This was so time-based operations like windowing, joining, or aggregating could remain consistent. The ZonedDateTime approach (which is superior and we should move to), would require a `.withZoneSameInstant(ZoneId.systemDefault())` for the same normalization, but we end up in locality quirks are you were observing. I'm not 100% sure if our assumption about date normalization is still valid and watermarking quirks can be very insidious, but it's been awhile since I've looked at it. Perhaps @mbroecheler has some input, or I can take a closer look at it later today.